packages/acgtk/acgtk.1.5.1/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "sylvain.pogodalla@inria.fr"
+
+build: [
+  ["dune" "subst"] {pinned}
+# remove the -p to also build the local libraries: conflict with the
+# fact that some libraries are also part of the acgtkLib package
+#  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "-j" jobs]
+]
+
+install: ["dune" "install"]
+
+depends: [
+  "ocaml" { >= "4.05.0"}
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.1"}
```
Dune should not be tagged as a build dependency (see #14266) and your `dune-project` file says this package requires at least dune 1.1